### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by Peter-developer01** [Peter-developer01/Peter-developer01#4] [Q&A]
Write here any questions for me.

**Comment by nchistov** [comment]
Have you got Google account? I have, and if you have too we can talk in Google.Chat.

**Reply by Peter-developer01** [reply]
No, I don't have Google account, sorry. :disappointed:
But we have an Stack Exchange Chat. I'm not sure, but probably mods can create an private room for us.

**Reply by nchistov** [reply]
Who is "mods"?

**Reply by Peter-developer01** [reply]
Moderators.

**Reply by nchistov** [reply]
No, I think we needn't create a private chat room, we can talk there, on GitHub!

**Reply by Peter-developer01** [reply]
Haha, nice!

**Comment by nchistov** [comment]
Oh, thank you for smart2fa project! Now I can use github! ♥

**Reply by Peter-developer01** [reply]
Your brother said that too :smile: 
You are welcome!

*It's not my project.

**Reply by nchistov** [reply]
I know that is not your project.

**Comment by Peter-developer01** [comment]
@nchistov Do you have CodeWars profile? If not, please use this link: www.codewars.com/r/J_s_qg , it'll give me additional honor.

**Reply by nchistov** [reply]
I haven't got CodeWars, and this link redirect me to home page.

**Reply by Peter-developer01** [reply]
If you register, it'll give me "reputation". (Use the updated link)

**Reply by nchistov** [reply]
@Peter-developer01 You should to earn reputation yourself. :)

**Reply by Peter-developer01** [reply]
@nchistov CodeWars is different from SO!

**Reply by Peter-developer01** [reply]
It is called "honor", not "reputation". And do you know "referral links"?

**Reply by Peter-developer01** [reply]
@nchistov  here

**Reply by nchistov** [reply]
@Peter-developer01 No I don't know what referral links is.

**Reply by nchistov** [reply]
@Peter-developer01 When I will register CodeWars (not now), I use this link.

**Reply by Peter-developer01** [reply]
@nchistov You can view this: https://ru.wikipedia.org/wiki/%D0%A0%D0%B5%D1%84%D0%B5%D1%80%D0%B0%D0%BB

**Reply by nchistov** [reply]
@Peter-developer01 thank you, now I know what referral links is!

**Comment by nchistov** [comment]
Gitflic 2.16.0 is out! It views cool now!

**Comment by nchistov** [comment]
Now I had Linux Mint 21.2, not windows! 🐧

**Reply by Peter-developer01** [reply]
Super!

**Comment by nchistov** [comment]
Do you know a project that transforms image to only-lines drawing? I can't find it.

**Reply by Peter-developer01** [reply]
Hmm? For example?

**Reply by Peter-developer01** [reply]
http://linify.me/

Something like that?

**Reply by Peter-developer01** [reply]
![image](https://github.com/Peter-developer01/Peter-developer01/assets/101456132/41c40528-207c-4a4a-8dda-67bcf8e68f09)

**Reply by nchistov** [reply]
Yes, but only black lines.

**Reply by Peter-developer01** [reply]
![linify_2024-01-05_15-56](https://github.com/Peter-developer01/Peter-developer01/assets/101456132/0524fb70-4207-4b5b-aa43-2245a75a9ac1)

Linify has an "Grayscale" option.

![image](https://github.com/Peter-developer01/Peter-developer01/assets/101456132/2fb4d418-899d-4434-9799-9e45425aadf3)

**Reply by nchistov** [reply]
@Peter-developer01 ok, thank you!
